### ISSUE TITLE
Incomplete package on PyPI

### ISSUE THREAD

**Issue by tleonardi** [bihealth/altamisa#58]
* AltamISA version: 0.2.0
* Python version: 3.7
* Operating System: Linux

### Description
Only the parent `altamisa` package is packaged by setuptools, whereas `altamisa.apps`, `altamisa.isatab`, `altamisa.constants` are not.
The issue seems to be due to this line in `setup.py`:
```
packages=find_packages(include=['altamisa'])
```
which should instead be:
```
packages=find_packages(include=['altamisa*'])
```
I'd also like to use this as an opportunity to suggest [poetry](https://github.com/sdispater/poetry), which really simplifies packaging, publishing and dependency management. 

I spotted this issue as part of the JOSS review process (openjournals/joss-reviews#1610).

### What I Did

```
$ virtualenv -p python3.7 venv                                                                                                                                                                                                                                                              
Running virtualenv with interpreter /usr/bin/python3.7                                                                                                                                                                                                                                      
Using base prefix '/usr'                                                                                                                                                                                                                                                                    
New python executable in venv/bin/python3.7                                                                                                                                                                                                                                                 
Also creating executable in venv/bin/python                                                                                                                                                                                                                                                 
Installing setuptools, pip, wheel...                                                                                                                                                                                                                                                        
done. 
                                                                                                                                                                                                                                                   
(venv)  $ pip install altamisa                                                                                                                                                                                                                                                              
Collecting altamisa                                                                                                                                                                                                                                                                         
Collecting attrs (from altamisa)                                                                                                                                                                                                                                                            
  Using cached https://files.pythonhosted.org/packages/23/96/d828354fa2dbdf216eaa7b7de0db692f12c234f7ef888cc14980ef40d1d2/attrs-19.1.0-py2.py3-none-any.whl                                                                                                                                 
Installing collected packages: attrs, altamisa                                                                                                                                                                                                                                              
Successfully installed altamisa-0.2.0 attrs-19.1.0          
                                                                                                                                                                                                                                
(venv)  $ python                                                                                                                                                                                                                                                                            
Python 3.7.3 (default, Jun 24 2019, 04:54:02)                                                                                                                                                                                                                                               
[GCC 9.1.0] on linux                                                                                                                                                                                                                                                                        
Type "help", "copyright", "credits" or "license" for more information.                                                                                                                                                                                                                      
>>> from altamisa import isatab                                                                                                                                                                                                                                                             
Traceback (most recent call last):                                                                                                                                                                                                                                                          
  File "<stdin>", line 1, in <module>                                                                                                                                                                                                                                                       
ImportError: cannot import name 'isatab' from 'altamisa' (venv/lib/python3.7/site-packages/altamisa/__init__.py)                                                                                                                                                                            
>>> quit()                 
                                                                                                                                                                                                                                                                 
(venv)  $ ls venv/lib/python3.7/site-packages/altamisa   
exceptions.py  __init__.py  __pycache__  _version.py
```

**Comment by holtgrewe** [comment]
Hi, thanks for the suggestions. I'll address this with a `v0.2.1` release.

Thanks for suggesting poetry. My impression was always that it is not ready yet and I was happy with twine+versioneer so far. I'll put this on my list for the future, though.

**Comment by holtgrewe** [comment]
Fixed in `v0.2.1`, also on PyPI.
